mithril-aggregator/src/store/verification_key_store.rs

```diff
@@ -110,6 +110,9 @@ mod tests {
                     Signer {
                         party_id: party_id.clone(),
```

Comment:
In this `entities::Signer` struct the `party_id` is not optional (and it will be removed when we enforce key certification).
The `party_id` is optional when doing the wrapped key registration though.